app/templates/view_projects.html

```diff
@@ -0,0 +1,71 @@
+{% extends "layout.html" %}
+
+{% block title %} View Projects {% endblock %}
+
+{% block scripts %}
+<script type="text/javascript" src="{{ url_for('static', filename='scripts/table_sort.js') }}"></script>
+
+<script>
+function clearForm(form)
+{
+    for (var i = 0; i < form.elements.length; i++)
+    {
+        if (form.elements[i].type == "text")
+        {
+            form.elements[i].value = "";
+        }
+    }
+    form.submit();
```

Comment:
`form.submit()` should replaced with the code in view_times.html (see my below comment).